### PR TITLE
Updated MinimizedRoomCard with grid and deafen button

### PR DESCRIPTION
**Pull request by HonbraDev**
Storybook
image

Figma
image

### CODE REVIEW

**Review by wes337 — APPROVED** [review]

File: kibbeh/src/ui/MinimizedRoomCard.tsx [hunk 1]
```diff
@@ -29,19 +29,19 @@ export const MinimizedRoomCard: React.FC<MinimizedRoomCardProps> = ({
   room,
 }) => {
   const router = useRouter();
-
+  // gap-n only works with grid
```
Comment (overlisted) [inline kibbeh/src/ui/MinimizedRoomCard.tsx:32]: No it doesn't? We have flexbox by default, and `gap` is universal between flex and grid

Comment (HonbraDev) [reply kibbeh/src/ui/MinimizedRoomCard.tsx:32]: https://tailwindcss.com/docs/gap

Comment (overlisted) [reply kibbeh/src/ui/MinimizedRoomCard.tsx:32]: https://css-tricks.com/almanac/properties/g/gap/

Comment (HonbraDev) [reply kibbeh/src/ui/MinimizedRoomCard.tsx:32]: Overlisted looking at TailWind: Is this CSS?

Comment (HonbraDev) [reply kibbeh/src/ui/MinimizedRoomCard.tsx:32]: Wait a minute....

Chromium: 
image

Safari: 
Snímek obrazovky 2021-03-26 v 9 43 06

I am sorry Overlisted

**Pull request merged**